app/units/forms/enterprise_fees_bulk_update.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+module Forms
+  class EnterpriseFeesBulkUpdate
+    include ActiveModel::Model
+
+    validate :check_enterprise_fee_input
+    validate :check_calculators_compatibility_with_taxes
+
+    def initialize(params)
+      @errors = ActiveModel::Errors.new self
+      @params = params
+    end
+
+    def save
+      return false unless valid?
+
+      @enterprise_fee_set = Sets::EnterpriseFeeSet.new(@params)
+      @enterprise_fee_set.save
+      true
```

Comment:
We've lost the ability to know if `Sets::EnterpriseFeeSet.save` succeeded as we return true regardless of what  `Sets::EnterpriseFeeSet.save` returns. 
We need to check what `Sets::EnterpriseFeeSet.save` returns and if false add the errors to `EnterpriseFeesBulkUpdate` so that the controller can deal with them if necessary.